test_helpers/pool_helper.go

```diff
@@ -206,6 +208,45 @@ func SetInstanceRO(ctx context.Context, dialer tarantool.Dialer, connOpts tarant
 		return err
 	}
 
+	checkRole := func(conn *tarantool.Connection, isReplica bool) string {
+		data, err := conn.Do(tarantool.NewCallRequest("box.info")).Get()
+		switch {
+		case err != nil:
+			return fmt.Sprintf("failed to get box.info: %s", err)
+		case len(data) < 1:
+			return "box.info is empty"
+		}
+
+		status, statusFound := data[0].(map[interface{}]interface{})["status"]
+		readonly, readonlyFound := data[0].(map[interface{}]interface{})["ro"]
```

Comment:
Please, add type conversion checks to avoid panics on runtime.

```Go
if d, ok := data[0].(map[interface{}]interface{}); !ok {
    ...
}
```

Reply:
Done.
Also the same problem was fixed in 'pool' package.